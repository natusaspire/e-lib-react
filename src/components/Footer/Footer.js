import React, { memo } from 'react';

const Footer = () => <div className="footer" />;

export default memo(Footer);
