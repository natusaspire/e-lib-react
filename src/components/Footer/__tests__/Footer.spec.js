import React from 'react';
import { shallow } from 'enzyme';

import Footer from '@/components/Footer';

describe('<Footer />', () => {
  it('should render footer content', () => {
    const wrapper = shallow(<Footer />);

    expect(wrapper).toMatchSnapshot();
  });
});
