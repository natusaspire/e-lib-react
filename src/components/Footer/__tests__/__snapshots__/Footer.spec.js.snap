// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render footer content 1`] = `
<div
  className="footer"
/>
`;
